olive/cli/capture_onnx.py

```diff
@@ -207,7 +218,7 @@ def get_run_config(self, tempdir: str) -> Dict:
 
 
 TEMPLATE = {
-    "input_model": {"type": "HfModel"},
+    "input_model": {"type": "HfModel", "load_kwargs": {"trust_remote_code": True}},
```

Comment:
I don't think we want to force this to always be true. There are various undesirable scenarios with this:
- user should opt in since the remote code might be malicious
- for models like phi3, it forces the model to load using local code even though the model type is already supported directly by transformers 

trust_remote_code is already an option for hf_model group https://github.com/microsoft/Olive/blob/625161942dedf2660b6b53714149a4019fd06ea2/olive/cli/base.py#L101 so it can be added to the model config like in https://github.com/microsoft/Olive/blob/625161942dedf2660b6b53714149a4019fd06ea2/olive/cli/finetune.py#L200

Reply:
Oh i didn't realize it is one of the model options. I'll update this.